src/amo/components/AddonMeta/index.js

```diff
@@ -87,10 +74,6 @@ export class AddonMetaBase extends React.Component<InternalProps> {
         <MetadataCard
           className="AddonMeta-overallRating"
           metadata={[
-            {
-              content: userCount,
-              title: userTitle,
-            },
```

Comment:
Hi @twisted-sres thanks for your PR, maintainers are on vacation. But as I see by doing so , we are removing the user counts for every addon on AMO which is not the requested in issue. Actually we need to hide `No users` for the addons related to `Search Tools`, for example for this addon localhost:3000/en-US/firefox/addon/youtube-10423/ we need to hide `No users`